### DISCUSSION TITLE
How to use the new Grammer of Graphics lets-plot in pyshiny?

### DISCUSSION THREAD

**Discussion by pevolution-ahmed** [posit-dev/py-shiny#1589] [Q&A] [answered]
Hello everyone,

Finally I found a new plotting library that was influenced by ggplot2 and really good at implementing Grammer of graphics principales.

What make it shines is the interactivity that's similar to plotly.

Here is a link to the lib:

https://lets-plot.org/python/index.html

I'd like to know if it would work with shiny or not? Since its working on Jupiter notebooks

**Comment by cpsievert** [comment, marked as answer]
See [this thread](https://github.com/posit-dev/py-shiny/issues/448) for working examples
